### PR TITLE
Fix: Grid view - tab switch bug

### PR DESCRIPTION
**Pull request by pranavxc**
## Change Summary

re #4402

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Comment by wingkwong** [comment]
Please rebase and resolve conflict

**Comment by pranavxc** [comment]
> Please rebase and resolve conflict

Done

File: packages/nc-gui/components/smartsheet/Grid.vue [hunk 1]
```diff
@@ -707,7 +712,7 @@ watch(
                     @mouseover="selectBlock(rowIndex, colIndex)"
                     @contextmenu="showContextMenu($event, { row: rowIndex, col: colIndex })"
                   >
-                    <div class="w-full h-full">
+                    <div class="w-full h-full" v-if="!switchingTab">
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/Grid.vue:715]: please lint once. I believe `v-if` will be placed before `class`

**Review by wingkwong — APPROVED** [review]

**Review by dstala — APPROVED** [review]

**Comment by dstala** [comment]
@mertmit merging this PR sans your review. Inform author if you have review comments still

**Pull request merged**